slrealizer/utils/utils.py

```diff
@@ -1,23 +1,58 @@
-#==============================================
-
+#import numpy as np
+#from fractions import Fraction
+#import math
 import numpy as np
-import pandas
-import desc.slrealizer
-from fractions import Fraction
-import math
+#import pandas as pd
+
+def e1e2_to_ephi(e1, e2):
+    e = np.power(np.power(e1, 2.0) + np.power(e2, 2.0), 0.5)
+    phi = np.arctan(e2/e1)
+    return e, phi
+
+def ephi_to_e1e2(e, phi):
+    e1 = e*np.cos(phi)
```

Comment:
These `phi`'s should be `2*phi`'s, I think

Reply:
Done